requirements.yml

```diff
@@ -0,0 +1,13 @@
+name: haddock3
+channels:
+    - conda-forge
+    - defaults
+dependencies:
+    - python==3.8
+    - pip
+    - numpy
+    - toml
+    - jsonpickle
+    - pip:
+        - pdb-tools
+        - lightdock
```

Comment:
I would not list lightdock as a dependency, you don't need it to run haddock.

Reply:
It is listed as a dependency in the current version. I will not remove in this PR. We can discuss it in a separate issue.